april_tag_updates/docking_script.py

```diff
@@ -0,0 +1,127 @@
+expected_dist = 37
+known_width = 4.18 ##inches
+focal_length = 1.37478492e+03
+from electrical.motor_controller import BasicMotorController
+import numpy as np
+import cv2
+import time 
+from enum import Enum
+
+arucoDict = cv2.aruco.Dictionary_get(cv2.aruco.DICT_APRILTAG_36h11)
+arucoParams = cv2.aruco.DetectorParameters_create()
+cap = cv2.VideoCapture(0)
+window = 'Camera'
+cv2.namedWindow(window)
+def distance_to_camera(knownWidth, focalLength, perWidth):
+  # compute and return the distance from the maker to the camera
+  return (knownWidth * focalLength) / perWidth
+
+def direction_of_tag(corner,image):
+  (_, _, _, _) = corner
+  _,w = image.shape[0], image.shape[1]
+  cx, _ = int(960),int(540)
+  if  bottomLeft[0] > cx:
+    return "right"
```

Comment:
nit, enums might be better

Reply:
You use enums when the meaning of a variable is ambiguous. its not here. I think we good :))))

Reply:
hmm i think saying enums convey meanings when ambiguous is only one aspect of the benefits for enums.

Imo enums are labels for categories. Strings can be anything Let's say someone uses another func that returns "Left" instead of "left", then your equality check is invalid. Or someone can return "down" or change your code in the future thinking it's another direction since strings can be anything. Enums make the possibilities finite, explicitly conveyed, and error prone + compiler checking benefits (e.g. switch cases, check if valid direction).